torch_np/_wrapper.py

```diff
@@ -802,10 +905,15 @@ def argsort(a, axis=-1, kind=None, order=None):
 
 @asarray_replacer()
 def angle(z, deg=False):
+#    z = asarray(z, dtype='float64').get()
```

Comment:
Delete?

Reply:
Working through this function made me realize a fun bug with doing ndarray dunders: `__sub__ = __rsub__ = <ufunc.sub>` and all other non-commutative functions. This form makes `2 - np.array(4)` equal 2 not -2 :-). Will fix separately.

Reply:
Yep, you need to apply the operation on the reversed arguments :D